client.go

```diff
@@ -389,6 +392,10 @@ func (c *ovndb) Close() error {
 	return nil
 }
 
+func (c *ovndb) GetSchema() (libovsdb.DatabaseSchema, error) {
+        return c.client.Schema[c.db], nil
```

Comment:
@Pardhakeswar  I don't think you need to return error at all.. since it is going to be `nil` all the time

Reply:
Updated it @girishmg